test/integration/vistor_can_view_items_test.rb

```diff
@@ -13,6 +13,6 @@ def create_items(num)
     visit items_path
 
     # adding within css not working
-    assert page.has_content?('latte')
+    assert page.has_content?('Latte')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.